src/basic/unit-def.h

```diff
@@ -5,14 +5,18 @@
 
 #include "macro.h"
 
+/* The order is relevant to match subsets:
```

Comment:
Maybe say """
The order is used to order unit jobs by priority:
- exec type units that fork new processes (lowest priority)
- ...
- ... (highest priority)
"""

Reply:
I've changed it like this. There are only two groups now, because for sorting, sockets are not treated separately.